tensorflow/g3doc/hardware/adding_support/index.md

```diff
@@ -0,0 +1,121 @@
+# Adding Support to New Hardware in TensorFlow
+
+New compute hardware is be release frequently and 
```

Comment:
For the purposes of this document, I would have this section just say:

"This is a work-in-progress document for how one might add support for custom hardware to TensorFlow.
It is assumed that you can link C/C++ code that uses your particular hardware into TensorFlow."